### DISCUSSION TITLE
Just curious about cpeditor-bin AUR packages

### DISCUSSION THREAD

**Discussion by UmeshKonduru** [cpeditor/cpeditor#1410] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/948d402c-30d7-4e44-85e1-93b96f5ed909)

I don't see the `cpeditor-bin` package mentioned either on the website or the README, but it seems to have been added quite recently and the last update was only 10 days ago. What's the difference between `cpeditor-bin` and the other two packages?

**Comment by ouuan** [comment, marked as answer]
`cpeditor-bin` is not officially maintained. See the different maintainers. It extracts the contents of the deb package, which uses bundled libraries instead of system libraries.
